pkg/apis/custom_metrics/doc.go

```diff
@@ -0,0 +1,17 @@
+// Copyright 2017 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +k8s:deepcopy-gen=package,register
+
+package custom_metrics // import "k8s.io/metrics/pkg/apis/custom_metrics"
```

Comment:
You'll need to specify a group name.  See https://github.com/kubernetes/sample-apiserver/blob/master/pkg/apis/wardle/doc.go#L20 as an example.